tests/e2e/utils/utils.js

```diff
@@ -173,7 +173,9 @@ async function waitForValidOutputFile(page) {
 }
 
 async function waitAndClick(page, id) {
-  await page.waitForSelector(id);
+  await page.waitForSelector(id, {
+    timeout: 2000
```

Comment:
AFAIK the default value is 30000...
https://pptr.dev/#?product=Puppeteer&version=v5.2.1&show=api-pagewaitforselectorselector-options

Reply:
@odeimaiz , you are totally right. I misread that the default was 1s